--- a/gui_app/ui.c
+++ b/gui_app/ui.c
@@ -10,6 +10,7 @@
 #include "./pages/ui_GameMemoryPage/ui_GameMemoryPage.h"
 #include "./pages/ui_DrawPage/ui_DrawPage.h"
 #include "./pages/ui_CalculatorPage/ui_CalculatorPage.h"
+#include "./pages/ui_MeetingDemoPage/ui_MeetingDemoPage.h"
 ///////////////////// VARIABLES ////////////////////
 
 lv_lib_pm_t page_manager;
@@ -25,5 +26,5 @@
 ///////////////////// all apps ////////////////////
 
-#define _APP_NUMS 11 // number of apps (including HomePage)
+#define _APP_NUMS 12 // number of apps (including HomePage)
 
 ui_app_data_t ui_apps[_APP_NUMS] = 
@@ -104,7 +105,13 @@
         .name = "CalculatorPage",
         .init = ui_CalculatorPage_init,
         .deinit = ui_CalculatorPage_deinit,
         .page_obj = NULL
-    }
+    },
+    {
+        .name = "MeetingDemoPage",
+        .init = ui_MeetingDemoPage_init,
+        .deinit = ui_MeetingDemoPage_deinit,
+        .page_obj = NULL
+    }
 
 };
